### ISSUE TITLE
For reference, Fontra’s version 

### ISSUE THREAD

**Issue by justvanrossum** [behdad/models.js#1]
https://github.com/googlefonts/fontra/blob/main/src/fontra/client/core/var-model.js

**Comment by justvanrossum** [comment]
https://github.com/googlefonts/fontra/blob/main/test-js/test-var-model.js

**Comment by behdad** [comment]
Yep, it didn't occur to @lorp and I that you probably have done it already... :)

**Comment by behdad** [comment]
We found after the fact.

**Comment by justvanrossum** [comment]
Should we somehow combine and publish on npm?

**Comment by behdad** [comment]
> Should we somehow combine and publish on npm?

Yes please.

I prefer my version because it's closer to the Python version, so porting changes would be easier, but I let you combine. My JS is zero...
